README.md

```diff
@@ -90,7 +90,155 @@ Commit this file to your Git repository as it ensures future builds to use the v
+There are three ways to test a package:
+
+1. [Make a standalone setup and mock the API responses](#make-a-standalone-setup)
+2. [Proxy the MOLGENIS backend in front of the package](#proxying-molgenis-backend)
+3. [Link the package in the MOLGENIS backend for full integration](#using-yarn-link-for-live-editing) 
+
+> **note: the ordering of the ways to test the packages is important.**
+>
+>The first and second test manners should be sufficient to actually be confident that your application works. In the preview mode, 
+where you already integrated the artifact in MOLGENIS you also test the integration. 
+
+#### Make a standalone setup
+You always want to start with an offline setup for the package. You then need to define the state and mock 
+responses first and have to think about the API you want to use. In cli2 and cli3 the setup is different so 
+both are described.  
+
+##### cli2
+When you are creating a standalone setup in cli2 you need to add an ```index.html``` file (is generated automatically).
+Besides this you need to add mock responses to supply you **store**. You can add them in ```#package#/config/index.js```. 
+
+*Example:*
+
+```javascript
+...
+const listOfItems = require('./dev-responses/list.js')
+const localizedMessages = require('./dev-responses/localized-messages.js')
+const localizedFormMessages = require('./dev-responses/localized-ui-form-messages')
+...
+  before (app) {
+      app.get('/menu/plugins/#package#/list', function (req, res) {
+        res.json(listOfItems)
+      })
+      app.get('/api/v2/i18n/#package#/en', function (req, res) {
+        res.json(localizedMessages)
+      })
+      app.get('/api/v2/i18n/ui-form/en', function (req, res) {
+        res.json(localizedFormMessages)
+      })
+...
+```
+
+This before block is used by the **store** which instead of accessing the real end-points. This is also used by the end-to-end tests to test the ui.
+
+##### cli3
+When you are creating a standalone setup in cli3 you need to add an ```index.html``` file (is generated automatically).
+Besides this you need to add mock responses to supply you **store**. You can add them in ```vue.config.js```.
+ 
+*Example:* 
+ 
+```javascript
+...
+const listOfItems = require('./dev-responses/list.js')
+const localizedMessages = require('./dev-responses/localized-messages.js')
+const localizedFormMessages = require('./dev-responses/localized-ui-form-messages')
+...
+  before: function (app, server) {
+      app.get('/menu/plugins/#package#/list', function (req, res) {
+        res.json(listOfItems)
+      })      
+      app.get('/api/v2/i18n/#package#/en', function (req, res) {
+        res.json(localizedMessages)
+      })
+      app.get('/api/v2/i18n/ui-form/en', function (req, res) {
+        res.json(localizedFormMessages)
+      })           
+...
+``` 
+
+This before block is used by the **store** which instead of accessing the real end-points. This is also used by the end-to-end tests to test the ui.
+ 
```

Comment:
I think it would be nice to finish this section with an example. Something like:

1. Run `yarn this and that` to run your package
2. Look _here and there_ to see that your data is being mocked.

Reply:
Because right now I'm not sure how to confirm that my actions were successful.